### PR TITLE
Prevent the error (in console) from forwarded motions

### PR DESCRIPTION
**Pull request by Elblinator**
part of #3605
I still cannot reproduce the other part of the bug but I can reliably reproduce the error in the console

### CODE REVIEW

**Review by MSoeb — APPROVED** [review]
Works as intented. No error in console.

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.ts [hunk 1]
```diff
@@ -353,8 +353,12 @@ export class MotionMetaDataComponent extends BaseMotionDetailChildComponent impl
...
                 .pipe(
                     map(motion => [
-                        motion.referenced_in_motion_recommendation_extensions,
-                        motion.recommendation_extension_references as ViewMotion[]
+                        motion.referenced_in_motion_recommendation_extensions
+                            ? motion.referenced_in_motion_recommendation_extensions
+                            : null,
+                        motion.recommendation_extension_references
+                            ? (motion.recommendation_extension_references as ViewMotion[])
+                            : null
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.ts:361]: I don't see how this should fix the issue. 

Trying to read a property of `null` will still result in throwing a type error. The error message in the issue looks to me like `motion` in line 355 is null. Which also does not really make sense. 

If it is an empty `motion` attribute the solution should look like this:
```suggestion
                        motion?.referenced_in_motion_recommendation_extensions,
                        motion?.recommendation_extension_references as ViewMotion[]
```

**Review by bastianjoel — APPROVED** [review]
If you disagree with my last comment I am okay with merging this. I did not look to much into this myself.

**Comment by Elblinator** [comment]
Both solutions seem to prevent the Error and yours is shorter, so I would prefer that one.

**Review by MSoeb — APPROVED** [review]

**Pull request merged**